app/screens/post_options/index.ts

```diff
@@ -51,7 +58,22 @@ const canEditPost = (isOwner: boolean, post: PostModel, postEditTimeLimit: numbe
     return cep;
 };
 
-const enhanced = withObservables([], ({post, showAddReaction, location, database}: WithDatabaseArgs & { post: PostModel; showAddReaction: boolean; location: string }) => {
+const withPost = withObservables([], ({post, database}: {post: Post | PostModel} & WithDatabaseArgs) => {
+    let id: string | undefined;
+    let combinedPost: Observable<Post | undefined> = of$(undefined);
+    if (post.type === Post.POST_TYPES.COMBINED_USER_ACTIVITY && post.props?.system_post_ids) {
+        const systemPostIds = post.props.system_post_ids as string[];
```

Comment:
I don't fully under the `system_post_ids`  format.  Will `this` post always be the last one?  What do the other part of the id represent?

Reply:
talked about this in a session ;)